registry/storage-drivers/filesystem.md

```diff
@@ -10,7 +10,8 @@ An implementation of the `storagedriver.StorageDriver` interface which uses the
 
 * `rootdirectory`: (optional) The absolute path to a root directory tree in which
 to store all registry files. The registry stores all its data here so make sure
-there is adequate space available. Defaults to `/var/lib/registry`.
+there is adequate space available. Defaults to `/var/lib/registry`. If the directory
+does not exist, it will be created with the `0777 &~ umask` permission.
```

Comment:
Noob question. Is `0777 &~ umask` an actual command or are you just trying to signal that the actual permission used is the result of doing a logic AND of `0777` and whatever `umask` is set with?

Reply:
It might be better to link to a doc on umask and say that this honors `umask`.

Reply:
updated, ptal